packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/LogoInput/index.js

```diff
@@ -0,0 +1,116 @@
+import React, { useReducer } from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { CarouselInput, CarouselSlide, CarouselActions } from '@strapi/design-system/CarouselInput';
+import { IconButton } from '@strapi/design-system/IconButton';
+import { Box } from '@strapi/design-system/Box';
+import Plus from '@strapi/icons/Plus';
+import Refresh from '@strapi/icons/Refresh';
+import reducer, { initialState } from './reducer';
+import LogoModalStepper from '../LogoModalStepper';
+import { SIZE, DIMENSION } from '../../utils/constants';
+import stepper from './stepper';
+
+const LogoInput = ({ customLogo, defaultLogo, onChangeLogo, onResetMenuLogo }) => {
+  const [{ currentStep }, dispatch] = useReducer(reducer, initialState);
+  const { Component, next, prev, modalTitle } = stepper[currentStep] || {};
+  const { formatMessage } = useIntl();
+
+  const goTo = to => {
+    dispatch({
+      type: 'GO_TO',
+      to,
+    });
+  };
+
+  return (
+    <>
+      <CarouselInput
+        label={formatMessage({
+          id: 'Settings.application.customization.carousel.title',
+          defaultMessage: 'Logo',
+        })}
+        selectedSlide={0}
+        hint={formatMessage(
+          {
+            id: 'Settings.application.customization.carousel-hint',
+            defaultMessage:
+              'Change the admin panel logo (Max dimension: {dimension}*{dimension}, Max file size: {size}KB)',
+          },
+          { size: SIZE, dimension: DIMENSION }
+        )}
+        previousLabel="Previous slide"
+        nextLabel="Next slide"
```

Comment:
hey, actually we already discussed that with Hicham there 
<img width="937" alt="image" src="https://user-images.githubusercontent.com/71838159/165524278-f19cf015-c056-4e09-a178-d6c22d7b7f03.png">
Those labels aren't used (since it will always be a single media) but are required

Reply:
we could replace it by empty strings though, I thought it didn't accept them, WDYT?

Reply:
What do you think about adding a code comment and making them empty strings? :)